src/platform_glfw/Renderer.cpp

```diff
@@ -244,6 +249,10 @@ namespace Renderer
       return false;
     }
 
+#ifdef __APPLE__
+      Misc::ProcessWindow(mWindow);
```

Comment:
Don't see why this is in Misc; you could just have it as an #ifdef APPLE function inside Renderer.

Reply:
I thought Misc would be an appropriate place to put platform specific functionality. I need to call the ShowTouchBar method which needs to remain in the .mm file because .mm informs the compiler that it is an Objective-C++ file. It won't compile if I just bring that whole function out of that file

Reply:
Renderer can be platform-specific (indeed if you look at it, it has a few W32-specific pieces, although limited); it just happens to be luck that most of the OpenGL backend is the same across the board.

Putting ShowTouchBar in Renderer::Open sounds perfectly reasonable to me.

Reply:
Well as I said ShowTouchBar has to live in TouchBar.mm or it won't compile because the compiler will treat render.cpp as a C++ file but ShowTouchBar is Objective-C++

Reply:
Yes but you can call it from the renderer - Misc::ProcessWindow is entirely unnecessary.